app/Models/Business.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Business extends Model
+{
+    protected $table = "businesses";
+    
+    /**
+    * The attributes that are mass assignable.
+    *
+    * @var array
+    */
+    protected $fillable = [
+        'name', 'email', 'phone', 'logo', 'description', 'count_follow', 'user_id'
+    ];
+    
+    /**
+     * Relationship promotion tables.
+     *
+     * @return \Illuminate\Database\Eloquent\Relations\HasMany
+     */
+    public function promotions()
+    {
+        return $this->hasMany('App\Models\Promotion');
+    }
+    
+    /**
+     * Relationship follow tables.
+     *
+     * @return \Illuminate\Database\Eloquent\Relations\HasMany
+     */
+    public function follows()
```

Comment:
`follows` is just a pivot table. In this case `promotion` belongs to many `users` via `follows` table. Please check the `BelongsToMany` relationship.